### PR TITLE
fix: Job IDs can't have spaces.

### PR DESCRIPTION
**Pull request by feanil**
Update the template to not have any spaces in the job name.

### CODE REVIEW

File: workflow-templates/add-new-issues-to-a-project.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
     types: [opened]
 
 jobs:
-  Add an issue to project:
+  Add_an_issue_to_project:
```
Comment (kdmccormick) [inline workflow-templates/add-new-issues-to-a-project.yml:13]:
```suggestion
  add-an-issue-to-project:
```
I have no source for this, but I think I'm used to seeing these written in snake-case.

**Review by kdmccormick — APPROVED** [review]
LGTM, one optional nit.

File: workflow-templates/add-new-issues-to-a-project.yml (hunk 1 above)
Comment (kdmccormick) [reply workflow-templates/add-new-issues-to-a-project.yml:13]: you're right Carlos. I updated my suggestion.

**Review by Carlos-Muniz — APPROVED** [review]
Groovy!

**Pull request merged**